src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int NUM_FIGURES = 6;
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[NUM_FIGURES];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < NUM_FIGURES; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            System.out.println("Figure: " + figures[i].getClass().getSimpleName()
+                    + ", area: " + figures[i].getArea()
+                    + " sq.units, color: " + figures[i].getColor());
+        }

```

Comment:
```suggestion
        for (int i = 0; i < NUM_FIGURES; i++) {
            if (i < NUM_FIGURES / 2) {
                figures[i] = figureSupplier.getRandomFigure();
            } else {
                figures[i] = figureSupplier.getDefaultFigure();
            }
            figures[i].draw;
        }

```

Reply:
Oh, right, I'm a little confused now, I replaced the solution with your hint, thanks!)